Recruit-Training/Advanced-Recruit-Training/Embedded-Teaser/Training/Test/TrainingTest.c

```diff
@@ -5,24 +5,43 @@
 void runTrainingTests()
 {
     // Run all the tests here using RUN_TEST()
+    RUN_TEST(test_EverythingValid);
+    RUN_TEST(test_EverythingInvalid);
+    RUN_TEST(test_OnlyLightsInvalid);
+    RUN_TEST(test_OnlyMotorsInvalid);
 }
 
 void test_EverythingValid()
 {
-
+    uint8_t data[3] = {0b00000000, 0b00000000, 0b00111100};
+    trainingTask(data);
+    TEST_ASSERT_EQUAL_UINT8(0b00000111, validData);
+    TEST_ASSERT_EQUAL_UINT8_ARRAY(data, outputArray, 3);
```

Comment:
You should use the test assert message functions and write something that will clearly explain what failed and what it's suppose to be. This will make it easier on the user when the test does fail.